hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -87,6 +87,12 @@
   public static final String CACHE_COMPACTED_BLOCKS_ON_WRITE_KEY =
       "hbase.rs.cachecompactedblocksonwrite";
 
+  public static final String CACHE_COMPACTED_BLOCKS_ON_WRITE_THRESHOLD_KEY =
```

Comment:
This is in bytes only right?

Reply:
Yes. I have added a comment mentioning that